client/templates/userprofile/profile.js

```diff
@@ -6,4 +6,53 @@ Template.profile.helpers({
 		var first = Meteor.user().realname.split(' ');
 		return first[0];
 	}
+});
+
+Template.profile.events({
+	'click .edit-bio': function() {
+		if (Session.get("gearclicked"))
+		{
+			var biography = $(".editable-bio").val();
+			var origin = $("<p id='biography'></p>");
+			origin.text(biography);
+			$(".editable-bio").replaceWith(origin);
+			$(".statusbio").remove();
+			Session.set("gearclicked", false);
+
+		}
+		else
```

Comment:
This file does not conform to indentation conventions for this project. Specifically, lines should be indented with two spaces, at each level of indentation.